pytorch_lightning/loops/fit_loop.py

```diff
@@ -51,13 +51,13 @@ def __init__(
         super().__init__()
         self.max_epochs = 1000 if (max_epochs is None and max_steps is None) else max_epochs
         self.min_epochs = 1 if (min_epochs is None and min_steps is None) else min_epochs
-        self.training_loop = TrainingEpochLoop(min_steps, max_steps)
-        self.validation_loop = EvaluationDataLoaderLoop()
+        self.epoch_loop = TrainingEpochLoop(min_steps, max_steps)
```

Comment:
There's a conflict here with progress tracking.

The progress dataclass for fit has `train`  and `val` attributes which then have epoch/batch.

https://github.com/PyTorchLightning/pytorch-lightning/blob/8bd7b1bdd7d3f723822e78908033cf0a6743713a/pytorch_lightning/trainer/progress.py#L165-L166

The previous name did fit this structure but the new one doesn't

Reply:
it was suggested to me in the other PRs that `self.validation_loop.epoch_loop` is clearer to understand than `self.validation_loop.validation_loop`, so this was changed and I was following this logic. Let's brainstorm tomorrow how we want to make names consistent between progress tracking, loops, and rest of code base.